### DISCUSSION TITLE
[Question] Does this support Nvidia 2000 gpus?

### DISCUSSION THREAD

**Discussion by jxnding** [xCuri0/ReBarUEFI#55] [General]
Is this for adding support to unsupported motherboards or GPUs too?

I've read through the README and looked through the supported mobos list, which seems to imply that this is just for hacking CPU/mobo support and not GPU support.

Would this work on my system?
5800X / B550 / 2080 Ti

**Comment by xCuri0** [comment]
@jxnding Only motherboards. You can force a BAR resize on Turing GPUs by adding an override to this module but I doubt it would work

**Reply by Black6spdZ** [reply]
NvStrapsRebar

**Comment by Sid127** [comment]
out of curiosity, what'd be the correct way to force a BAR resize?

**Comment by xCuri0** [comment]
@Sid127  See the 5600 XT override https://github.com/xCuri0/ReBarUEFI/blob/master/ReBarDxe/ReBar.c#L167

You should try doing these modifications in Linux kernel first though since its easier to test there. See https://elixir.bootlin.com/linux/latest/source/drivers/pci/pci.c#L3710

It could be possible that the VBIOS won't accept larger but you need to try and see. And you have a laptop so it would be technically possible to remove that limit but would need some advanced reverse engineering knowledge.

**Comment by terminatorul** [comment]
@jxnding @Sid127
 I also have 2080 Ti and very recently I could get this to work: https://github.com/terminatorul/NvStrapsReBar/. It still requires re-compiling the DXE driver (NvStrapsReBar.ffs) manually, but some people had positive results.
